rest/taskana-rest-spring/pom.xml

```diff
@@ -214,7 +214,12 @@
                             <groupId>commons-logging</groupId>
                         </exclusion>
                     </exclusions>
-                </dependency>                
+                </dependency>
```

Comment:
Why is this required? We already have this dependency:

                <dependency>
                        <groupId>pro.taskana</groupId>
                        <artifactId>taskana-spring</artifactId>
                        <version>${project.version}</version>
                </dependency>

Reply:
you are right